lib/gpio.js

```diff
@@ -221,7 +221,8 @@ GPIO.prototype._get = function(fn) {
 GPIO.prototype.set = function(v, fn) {
 	var self = this;
 	var callback = typeof v === 'function' ? v : fn;
-	if(typeof v !== "number" || v !== 0) v = 1;
+	if (typeof v == "boolean") v = (v) : 1 : 0;
```

Comment:
In this line, don't you mean `v = v ? 1 : 0;`?
Also, I'd use the identity operator (triple equal) for clarity.